Tests/Unit/Microsoft365DSC/Microsoft365DSC.IntuneAccountProtectionLocalUserGroupMembershipPolicy.ps1

```diff
@@ -0,0 +1,560 @@
+[CmdletBinding()]
```

Comment:
This file should be named Microsoft365DSC.IntuneAccountProtectionLocalUserGroupMembershipPolicy.Tests.ps1. Currently the .Tests is missing at the end of the file.